mne/source_space.py

```diff
@@ -1348,21 +1355,22 @@ def setup_source_space(subject, fname=True, spacing='oct6', surface='white',
 
 
 @verbose
-def setup_volume_source_space(subject, fname=None, pos=5.0, mri=None,
+def setup_volume_source_space(subject=None, fname=None, pos=5.0, mri=None,
                               sphere=(0.0, 0.0, 0.0, 90.0), bem=None,
                               surface=None, mindist=5.0, exclude=0.0,
-                              overwrite=False, subjects_dir=None,
+                              overwrite=None, subjects_dir=None,
                               volume_label=None, add_interpolator=True,
                               verbose=None):
     """Setup a volume source space with grid spacing or discrete source space.
 
     Parameters
     ----------
     subject : str
-        Subject to process.
+        Subject to process. If None, the path to the mri volume must be
+        absolute. Has no effect if pos is a dict.
```

Comment:
"Has no effect if pos is a dict"

This is no longer true, right? If it is still true, perhaps we should also set `subject_his_id` for discrete source spaces, too